core/src/main/java/org/moskito/control/plugins/psqlhistory/PSQLHistoryService.java

```diff
@@ -0,0 +1,186 @@
+package org.moskito.control.plugins.psqlhistory;
+
+import org.moskito.control.common.HealthColor;
+import org.moskito.control.common.Status;
+import org.moskito.control.config.MoskitoControlConfiguration;
+import org.moskito.control.core.Component;
+import org.moskito.control.core.ComponentRepository;
+import org.moskito.control.core.history.StatusUpdateHistoryItem;
+import org.moskito.control.core.history.service.HistoryService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Array;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class PSQLHistoryService implements HistoryService {
+    private static final Logger log = LoggerFactory.getLogger(PSQLHistoryService.class);
+    private final PSQLHistoryPluginConfig config;
+
+    public PSQLHistoryService(PSQLHistoryPluginConfig config) {
+        this.config = config;
+
+        try {
+            Class.forName("org.postgresql.Driver");
+        } catch (Exception e) {
+            log.error(e.getMessage(), e);
+            throw new RuntimeException(e.getMessage(), e);
+        }
+
+        createTable();
+    }
+
+    private void createTable() {
```

Comment:
you should use flyway migrations for this. Also take a look at DBService from ano-db it already supports migrations.
Also using SQL directly seems like very 2005, why not use jpa?